### ISSUE TITLE
Holding gesture not working properly on mobile.

### ISSUE THREAD

**Issue by PhilippeAudet-Stratejia** [unoplatform/uno#7130]
### Current behavior

When using the holding gesture, the event is only called after moving the pointer, and when moving further will then set the holding state to canceled. 
![HoldingAndroid](https://user-images.githubusercontent.com/83981176/133820697-8e148f21-3358-421b-bfd3-02cba8373664.gif)

On UWP, the gesture is working as intended.
![HoldingUWP](https://user-images.githubusercontent.com/83981176/133820705-fabcdd87-3d77-4484-8d44-22b52b8bb6a5.gif)
 

### Expected behavior

When holding down and not moving at all on mobile, the Holding event should be raised without having to move the cursor. Although it's hard to press completely still, it does happen and the intended behavior when holding isn't executed.

### How to reproduce it (as minimally and precisely as possible)

[HoldingSample.zip](https://github.com/unoplatform/uno/files/7187114/HoldingSample.zip)

On UWP : 
- Launch the project in simulator mode. 
- Select the basic touch input on the right pane.
- Press and hold on the blue rectangle. 
- Notice the text on top of the blue rectangle displaying the state of holding, which would be Started. 
- Move the cursor while still holding
- The state should go to canceled.

On Android/iOS: 
- Launch the project (It's easier to reproduce on simulator)
- Press down and hold without moving
- Notice how the holding state isn't "Started"
- While still holding, move a little in any direction. The state should then be "Started"
- Keep moving and the state will be canceled.

### Workaround

The holding still works, but it's not consistent. It can also be archieved manually with the pressed event and timers, but it's also not as consistent. 

### Works on UWP/WinUI

Yes

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia, Uno.UI.RemoteControl / Uno.WinUI.RemoteControl

### NuGet package version(s)

Uno.UI : 3.11.0-dev.113
Uno.RemoteControl : 3.11.0-dev.113
Uno.Core: 2.4.0

### Affected platforms

iOS, Android

### IDE

Visual Studio 2019

### IDE version

16.11.13

**Comment by dr1rrb** [comment]
This is the expected behavior: for perf consideration we don't start a timer on each pointer pressed to detect the "hold gesture" instead we rely on the fact that touches are imprecise and we continuously get a lot of pointer move when holding with a finger. So the "bug" is present only in the emulator where your are simulating fingers using a "precise" pointer (i.e. the mouse).

**Comment by jeromelaban** [comment]
@jeanplevesque @PhilippeAudet-Stratejia Can you validate if the same repro works on a physical device?

**Comment by PhilippeAudet-Stratejia** [comment]
I didn't think about that, but it's a good idea if it improves the perfs. As I said, it didn't happen often where I could stay completely still on a physical device, but i'll try again to see if the threshold good on the devices I own.

**Comment by jeromelaban** [comment]
@PhilippeAudet-Stratejia have you had the chance to validate ?

**Comment by dr1rrb** [comment]
@PhilippeAudet-Stratejia you mean the duration threshold? The issue here is that it's really subjective, last time that I've checked, I was found that it was easier to trigger the holding on uno than on UWP (both on touch devices obviously). If you were able to give me any measurable data/comparison it would be nice.

**Comment by PhilippeAudet-Stratejia** [comment]
I agree, this is really subjective and it's pretty hard to have any measurable data. 
With the current implementation, it's odd to have the HoldingStates Started and Canceled since, on UWP, the Canceled state is called when we move the cursor when holding, but on uno, we need to move to trigger the holding event. 

As for the event, I believe it's a matter of luck, as sometimes I found myself holding and wondering why the event isn't called, when in fact it's because I haven't moved enough yet.
